src/collective/excelexport/view.py

```diff
@@ -57,22 +56,29 @@ class ExcelExport(BaseExport):
     def _format_render(self, render):
         """Common formatting
         """
-        if isinstance(render, Message):
-            render = translate(render, context=self.request)
+        if isinstance(render, unicode):
+            return render
         elif isinstance(render, str):
-            render = unicode(render)
+            render = safe_unicode(render)
+        elif isinstance(render, Message):
+            render = translate(render, context=self.request)
         elif isinstance(render, DateTime):
```

Comment:
Are you sure you shouldn't check for isinstance of Message before the isinstance unicode? I think Message inherits from unicode.